xsuite/src/cli/newWalletCmd.ts

```diff
@@ -37,7 +40,7 @@ const action = async ({
   let keystore: Keystore;
   if (password === undefined) {
     try {
-      keystore = await Keystore.createFile(walletPath);
+      keystore = await Keystore.createFile(walletPath, undefined, shard);
```

Comment:
@lcswillems 

Is the a more elegant way in javascript to skip the arguments you dont want to set? (instead of writing undefined)

In C# you can do:
`Keystore.createFile(walletPath, shard: shard);`